libraries/AP_Networking/AP_Networking_port.cpp

```diff
@@ -217,12 +218,29 @@ void AP_Networking::Port::udp_server_loop(void)
     }
     sock->reuseaddress();
 
+#if AP_NETWORKING_UDP_SERVER_MULTI_CLIENT_ENABLED
+    if (type == NetworkPortType::UDP_SERVER_MULTI_CLIENT) {
+        udp_server_multi_client = NEW_NOTHROW UDP_Server_Multi_Client(sock, state.idx);
+        if (udp_server_multi_client == nullptr) {
+            GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "UDP[%u]: Failed to init multi-clients", (unsigned)state.idx);
+            delete sock;
```

Comment:
we don't often use delete
Do we normally tell the user via send-text if there's a failure?  I wonder if this is the best way or if we should log it instead

Reply:
this uses the same scheme as above it